about.html

```diff
@@ -0,0 +1,168 @@
+            class="flex bg-white text-[#272a31] px-[10%] py-[5%] text-[#746f6f] mx-[10%] border-solid border-[1px] border-[#d3d3d3]"
+          >
+            <p>
+              Come raise your glass with us on yet another one of our annual
+              parties where we share inside information with future Scientists
+              on how to brew the finest wine in the world,
+            </p>
+          </div>
+          <p class="text-black">
+            Communicate via Email for clarity and reservations!<br />
+            naledi22khosi@gmail.com
+          </p>
+        </div>
+      </section>
+      <hr class="bg-[#ec5242] flex w-[100%] item-center" />
+      <section
+        class="bg-white px-[10%] text-center text-black flex flex-col items-center gap-3"
+      >
+        <h2 class="text-center text-[#272a31] text-[25px] mt-[5%]">
+          The ultimate wine tasting party of 2023
+        </h2>
+        <hr class="w-[40px] bg-[#ec5242] h-[3px]" />
+        <p class="text-center">
+          The logo of CC Global Summit 2023 was decided through the logo
+          competition from 8, June to 7, July.
+        </p>
+        <div
+          class="my-[5%] px-[5%] py-[2%] border-solid border-[1px] border-[#d3d3d3]"
+        >
+          <img src="assets/wine-glass.png" alt="" />
+        </div>
+      </section>
+
+      <!-- past events -->
+
+      <hr class="bg-[#ec5242] flex w-[100%] item-center" />
+      <section
+        class="px-[10%] flex flex-col items-center gap-3 justify-center items-center"
+      >
+        <h2 class="text-center text-[#272a31] text-[25px] mt-[5%]">
+          See the past Wine tasting Parties
+        </h2>
+        <hr class="w-[40px] bg-[#ec5242] h-[3px]" />
+        <p>
+          Take a look at the past two Wine tasting parties which took place in
+          Lesotho and in Botswana
+        </p>
+        <div
+          class="ml-[3%] mt-[50px] pb-[100px] grid gap-[10px] grid-cols-1 md:grid-cols-2 justify-items-center"
+        >
+          <div
+            class="h-[320px] bg-[url('assets/hero-background.jpg')] bg-contain"
+          >
+            <div
+              class="text-[#fff] hover:bg-[#ec53423d] bg-[#ec5342bd] h-[inherit] p-[20px] flex justify-center flex-col flex-nowrap gap-5 items-center"
+            >
+              <h2 class="text-[25px] text-[#fff]">2019</h2>
+              <p class="text-center">Wine Tasting Party in Lesotho.</p>
+            </div>
+          </div>
+
+          <div
+            class="h-[320px] bg-[url('assets/hero-background.jpg')] bg-contain"
+          >
+            <div
+              class="text-[#fff] hover:bg-[#ec53423d] bg-[#ec5342bd] h-[inherit] p-[20px] flex justify-center flex-col flex-nowrap gap-5 items-center"
+            >
+              <h2 class="text-[25px] text-[#fff]">2021</h2>
+              <p class="text-center">Wine Tasting Party in Botswana.</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
```

Comment:
Kindly add the sponsor's section in the mobile version. This section is present in the provided template design, however, is missing in your case.  I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem.